### PR TITLE
Replaced a deprecated method.

### PR DESCRIPTION
**Pull request by m-agour**
Replaced deprecated method 'label' with 'vector_text'.
Also eliminated the "true division" runtime warning.

### CODE REVIEW

File: fury/actor.py [hunk 1]
```diff
@@ -1593,7 +1593,11 @@ def sphere(centers, colors, radii=1., phi=16, theta=16,
         sphere_actor.GetProperty().SetOpacity(opacity)
         return sphere_actor
 
-    scales = np.multiply(radii, [1, 1, 1])
+    if type(radii) in [float, int]:
+        scales = np.array([1, 1, 1]) * radii
+    else:
+        scales = np.array([1, 1, 1]) * radii[:, None]
```
Comment (xtanion) [inline fury/actor.py:1599]: This works perfectly fine @m-agour , 
You may also use `scales = np.array(radii)` as `repeat_primitive` handles the reshaping of the scales array.

**Review by xtanion — COMMENTED** [review]
Thanks, @m-agour for this PR. 
Please take a look at my comment below. Thanks

**Review by ganimtron-10 — COMMENTED** [review]
Hey @m-agour ,
All your changes are perfectly fine!!

There is just an issue with the sphere texture in `viz_earth_animation` which is inverted horizontally as shown below!!

earth_animation_issue
earth_animation_issue_2

File: fury/actor.py (hunk 1 above)
Comment (m-agour) [reply fury/actor.py:1599]: Hello @xtanion,
Thank you for reviewing my PR.
The compact solution you mentioned works just fine. Thanks for mentioning it.

**Comment by m-agour** [comment]
Hello @ganimtron-10,
Thank you for pointing out this issue. I'm looking into it.

**Review by skoudoro — APPROVED** [review]
Thank you for this @m-agour,

Overall, looks good to me. Good catch for the reverted earth @ganimtron-10! I think, this should be fixed in a new PR otherwise the title will not correspond to the change.

So, I will go ahead and merge it. Thanks all

**Pull request merged**